### ISSUE TITLE
[iOS] 1.0 crash report (iPhone 5)

### ISSUE THREAD

**Issue by ninjaprawn** [dzt/ftp#1]
Note: afaik 1.0.1 has been submitted, but here is the report anyway:

```
Incident Identifier: AC008870-47B8-4642-8E66-08142BA6530B
CrashReporter Key:   35ec4b69578ef6358481df07f0b6bf160b68216e
Hardware Model:      iPhone5,2
Process:             FTP [552]
Path:                /private/var/mobile/Containers/Bundle/Application/7CB640AB-213C-4992-A6A5-697E8E0FFCAE/FTP.app/FTP
Identifier:          com.kcufthepopulation.app
Version:             5 (1.0)
Code Type:           ARM (Native)
Parent Process:      launchd [1]

Date/Time:           2016-04-07 17:51:21.21 +1000
Launch Time:         2016-04-07 17:51:16.16 +1000
OS Version:          iOS 9.2 (13C75)
Report Version:      105

Exception Type:  EXC_BREAKPOINT (SIGTRAP)
Exception Codes: 0x0000000000000001, 0x00000000e7ffdefe
Triggered by Thread:  0

Filtered syslog:
None found

Global Trace Buffer (reverse chronological seconds):
0.709052     CFNetwork                  0x0000000023607e45 TCP Conn 0x14d811b0 SSL Handshake DONE
0.709580     CFNetwork                  0x0000000023607e45 TCP Conn 0x14d8f1d0 SSL Handshake DONE
1.588505     CFNetwork                  0x0000000023607d7f TCP Conn 0x14d8f1d0 starting SSL negotiation
1.588850     CFNetwork                  0x00000000236871a5 TCP Conn 0x14d8f1d0 complete. fd: 6, err: 0
1.590763     CFNetwork                  0x00000000236882a7 TCP Conn 0x14d8f1d0 event 1. err: 0
1.600118     CFNetwork                  0x0000000023607d7f TCP Conn 0x14d811b0 starting SSL negotiation
1.600694     CFNetwork                  0x00000000236871a5 TCP Conn 0x14d811b0 complete. fd: 5, err: 0
1.601063     CFNetwork                  0x00000000236882a7 TCP Conn 0x14d811b0 event 1. err: 0
2.280331     CFNetwork                  0x0000000023688325 TCP Conn 0x14d8f1d0 started
2.380634     CFNetwork                  0x0000000023688325 TCP Conn 0x14d811b0 started
2.477780     CFNetwork                  0x00000000236c8957 Creating default cookie storage with default identifier
2.477780     CFNetwork                  0x00000000236c8933 Faulting in CFHTTPCookieStorage singleton
2.477780     CFNetwork                  0x000000002370aaa7 Faulting in NSHTTPCookieStorage singleton

Thread 0 name:  Dispatch queue: com.apple.main-thread
Thread 0 Crashed:
0   FTP                             0x0003f240 0x21000 + 123456
1   FTP                             0x0003f300 0x21000 + 123648
2   UIKit                           0x272a5d78 -[UIViewController loadViewIfRequired] + 1028
3   UIKit                           0x272a5958 -[UIViewController view] + 24
4   UIKit                           0x27b1d948 -[_UIFullscreenPresentationController _setPresentedViewController:] + 72
5   UIKit                           0x275efa2c -[UIPresentationController initWithPresentedViewController:presentingViewController:] + 112
6   UIKit                           0x276132ce -[UIViewController _presentViewController:withAnimationController:completion:] + 3670
7   UIKit                           0x27615924 __62-[UIViewController presentViewController:animated:completion:]_block_invoke + 336
8   UIKit                           0x27615b90 -[UIViewController _performCoordinatedPresentOrDismiss:animated:] + 416
9   UIKit                           0x2739ef1e -[UIViewController presentViewController:animated:completion:] + 142
10  UIKit                           0x27618e26 -[UIViewController _showViewController:withAction:sender:] + 234
11  UIKit                           0x278bf778 __66-[UIStoryboardShowSegueTemplate newDefaultPerformHandlerForSegue:]_block_invoke + 128
12  UIKit                           0x279982d4 -[UIStoryboardSegueTemplate _performWithDestinationViewController:sender:] + 432
13  UIKit                           0x27998102 -[UIStoryboardSegueTemplate _perform:] + 66
14  UIKit                           0x2760af94 -[UIViewController performSegueWithIdentifier:sender:] + 80
15  FTP                             0x00041fb4 0x21000 + 135092
16  Alamofire                       0x001afc64 0x16d000 + 273508
17  Alamofire                       0x001b275c 0x16d000 + 284508
18  libdispatch.dylib               0x22cabdd6 _dispatch_call_block_and_release + 10
19  libdispatch.dylib               0x22cabdc2 _dispatch_client_callout + 22
20  libdispatch.dylib               0x22cb0670 _dispatch_main_queue_callback_4CF + 1532
21  CoreFoundation                  0x230d0fc4 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 8
22  CoreFoundation                  0x230cf4be __CFRunLoopRun + 1590
23  CoreFoundation                  0x23021bb8 CFRunLoopRunSpecific + 516
24  CoreFoundation                  0x230219ac CFRunLoopRunInMode + 108
25  GraphicsServices                0x2429baf8 GSEventRunModal + 160
26  UIKit                           0x2730dfb4 UIApplicationMain + 144
27  FTP                             0x0003ce48 0x21000 + 114248
28  libdyld.dylib                   0x22cd4872 start + 2
```

Not too sure if this is a server issue or Alamofire issue, but I'm just posting here so you know there is an issue. Also, this is just after app startup, so I see some stuff going on (glitchy FTP logo), but app crashes about 1.5s after.

**Comment by dzt** [comment]
This bug has been resolved in version 1.0.1, There was a problem with the buttons linking back to the storyboard on the ClosedVoewController due to the fact that the store was closed at the time of when you were trying to open the app, version 1.0.1 has been submitted to the App Store and should be in your hands mommenterly, sorry for the inconvenience.

**Comment by dzt** [comment]
Changlog for 1.0.1
- Closed Store Bug Fixed
- Small Design Enhancements
- Screen Optimization bug fixed for the iPhone 4s!"
